RUNBOOK.md

```diff
@@ -13,6 +13,20 @@ Please follow below instructions in order to upgrade the mediawiki version. Refe
 
 ## Deploying to prod
 
+1. Run the deploy script
+
+``` sh
+$ sudo SLACK_NOTIFICATIONS_URL=$(SLACK_NOTIFICATIONS_URL) /root/metakgp-wiki/scripts/deploy-latest.sh --go
```

Comment:
Why is `sudo` needed?

Reply:
The metakgp-wiki repository is in `/root` and it's not possible to run it without going into that directory. I want to move it to a separate group and move away from root deployments :sweat_smile: